lib/getFaustQueryResponse.js

```diff
@@ -0,0 +1,8 @@
+import { print } from '@apollo/client/utilities';
+import { sha256 } from 'js-sha256';
+
+export function useFaustQuery(query, props = null) {
+  const sha = sha256(print(query));
+
+  return props?.__FAUST_QUERIES__?.[sha];
+}
```

Comment:
Taken from the [original useFaustQuery function](https://github.com/wpengine/faustjs/blob/55c4e9fd345106ad0acdfae69b0db48a5616e3cd/packages/faustwp-core/src/hooks/useFaustContext.tsx#L15-L18)